src/platform/plugins/shared/dashboard/server/dashboard_saved_object/dashboard_saved_object.ts

```diff
@@ -43,6 +43,7 @@ export const createDashboardSavedObjectType = ({
       };
     },
   },
+  supportsAccessControl: true,
```

Comment:
Weren't we going to base this on a dashboard-specific feature flag instead? I am not sure we want to add the ability to specify this when registering a type quite yet, but let's discuss.